### ISSUE TITLE
Phase01-Sample

### ISSUE THREAD

**Issue by ahmadsalimi** [Star-Academy/codestar-intern-issues#41]
This issue template is based of [Phase01-Search.md](https://github.com/Star-Academy/codestar-internship/blob/master/Projects/Phase01-Search.md) contents.
Click on "Raw" button on right side of this page to see original markdown then create a new issue per team (two interns) in [this repo](https://github.com/Star-Academy/codestar-intern-issues/issues) with orginal content of tasklist below. Your issue title should be like this: `Phase01-Team5`

- [ ] Section 1: Think individually about question i & ii before starting
- [ ] Section 2: Read individually about Inverted-Index
- [ ] Section 3: Implement a simple inverted index with java (do pair programming)
  - [ ] Implement a fileReader class which reads all files in a folder, each file is a document in your inverted-index.
  - [ ] Implement an inverted-index class which contains all tokens and list of related documents per each token
  - [ ] Test your program with sample dataset mentioned in project definition
 
- [ ] Section4: Implement basic and, or, exclude features:
  - [ ] Support for words starting with + (or feature)
  - [ ] Support for words starting with nothing (and feature)
  - [ ] Support for words starting with - (exclude feature)
